tests/backup/backup_resiliency_test.go

```diff
@@ -949,21 +949,21 @@ var _ = Describe("{ScaleMongoDBWhileBackupAndRestore}", Label(TestCaseLabelsMap[
 			dash.VerifyFatal(err, nil, "Scaling backup MongoDB statefulset replica to original count")
 			log.Infof("mongodb replica after scaling back to original replica is %v", *statefulSet.Spec.Replicas)
 			dash.VerifyFatal(*statefulSet.Spec.Replicas == originalReplicaCount, true, "Verify mongodb statefulset replica after scaling back to original")
-			log.Infof("Verify that at least one mongodb pod is in Ready state")
+			log.Infof("Verify that at least two mongodb pod is in Ready state")
 			mongoDBPodStatus := func() (interface{}, bool, error) {
 				statefulSet, err = apps.Instance().GetStatefulSet(MongodbStatefulset, pxBackupNS)
 				if err != nil {
 					return "", true, err
 				}
 				if statefulSet.Status.ReadyReplicas < 2 {
-					return "", true, fmt.Errorf("no mongodb pods are ready yet")
+					return "", true, fmt.Errorf("minimum 2 mongodb pods are not ready yet")
```

Comment:
Better to print the number of pods that are ready. That way we will know if the ready pods were `0` or `1`

Reply:
done